src/components/UserInfo.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from "react";
+import api from "../queries/api";
+import { userFields } from "../queries/fields";
+
+const fields = userFields;
+class UserInfo extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { userInfo: "" };
+  }
+
+  componentDidMount() {
+    api
+      .user()
+      .getInfo()
+      .then(res => {
+        this.setState({ userInfo: res.data });
+      });
+  }
+
+  render() {
+    const { name, role, email } = fields.info;
+    const userInfo = this.state.userInfo;
+    const fullName = userInfo[name];
+    return (
+      <div>
+        {this.state.userInfo !== "" && (
+          <div className="btn-group">
+            <button
+              type="button"
+              className="btn btn-secondary dropdown-toggle btn-circle font-weight-bold"
+              data-toggle="dropdown"
```

Comment:
This must be using jquery. I know react and jquery can butt heads because they treat the DOM differently.

Reply:
Yep, things are implemented using the React way now (a component with an event listener).